airflow/contrib/hooks/datastore_hook.py

```diff
@@ -19,58 +19,81 @@
 #
 
 import time
+
 from googleapiclient.discovery import build
+
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
 
 
 class DatastoreHook(GoogleCloudBaseHook):
     """
-    Interact with Google Cloud Datastore. This hook uses the Google Cloud Platform
-    connection.
+    Interact with Google Cloud Datastore. This hook uses the Google Cloud Platform connection.
 
     This object is not threads safe. If you want to make multiple requests
     simultaneously, you will need to create a hook per thread.
     """
 
-    def __init__(self,
-                 datastore_conn_id='google_cloud_datastore_default',
-                 delegate_to=None):
+    def __init__(self, datastore_conn_id='google_cloud_datastore_default', delegate_to=None):
         super(DatastoreHook, self).__init__(datastore_conn_id, delegate_to)
-        self.connection = self.get_conn()
-        self.admin_connection = self.get_conn('v1beta1')
+        self.connection = None
+        self.admin_connection = None
 
-    def get_conn(self, version='v1'):
+    def get_conn(self, version='v1', is_admin=False):
```

Comment:
I think that this method should not have parameters. This will be consistent with other operators for GCP. Additional parameters can be passed by constructor.

Reply:
I am fine with changing this as well. This would also be a backwards incompatible change.

Reply:
Done.